src/resume-data.js

```diff
@@ -1,16 +1,30 @@
 const Resume = {
   header: {
```

Comment:
`header` object should be renamed to `basics`.

Reply:
okay.